### DISCUSSION TITLE
Default parameters

### DISCUSSION THREAD

**Discussion by Andro72** [kevalmorabia97/CoVA-Web-Object-Detection#13] [General]
Hi 
Can you provide parameter values for running main.py? It is a long list, and some of them are tricky to guess :-)

**Comment by kevalmorabia97** [comment]
You can check out the details for the arguments here: https://github.com/kevalmorabia97/CoVA-Web-Object-Detection/blob/master/utils.py#L15-L32

Default arguments should be good enough to run. You only need to specify the fold number (e.g. `--cv_fold 1`) out of the 5 cross validation folds

**Reply by Andro72** [reply]
Thanks. It is working :-)
